### ISSUE TITLE
Setting the root-object to null in the editor results in an error

### ISSUE THREAD

**Issue by thetruthger** [CarlosNZ/json-edit-react#90]
**Describe the bug**
Setting the root-object to `null` in the editor results in an error which logs:
```
index.esm.js:1 
 Uncaught 
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at ge (index.esm.js:1:57689)
    at gi (react-dom.production.min.js:160:137)
    at xs (react-dom.production.min.js:196:258)
    at Sl (react-dom.production.min.js:291:88)
    at bu (react-dom.production.min.js:279:389)
    at vu (react-dom.production.min.js:279:320)
    at gu (react-dom.production.min.js:279:180)
    at au (react-dom.production.min.js:270:88)
    at lu (react-dom.production.min.js:272:300)
```

**Expected behavior**
The expected behavior would be to either allow the root element to be null because null is still valid JSON, or at least set it back to what it was before without breaking. :)

**Screenshots**
![json_error](https://github.com/user-attachments/assets/1859c8b2-5510-44cb-8bf7-387bce5377d9)

**Online demo**
The error occurs also in the public online demo of this tool.

**Comment by CarlosNZ** [comment]
Thanks for pointing this out, will fix in the next few days 👍

**Comment by CarlosNZ** [comment]
Hi @thetruthger -- this should be fixed in [v1.13.3](https://www.npmjs.com/package/json-edit-react/v/1.13.3)
